weave/environment.py

```diff
@@ -158,13 +158,18 @@ def weave_wandb_cookie() -> typing.Optional[str]:
             raise errors.WeaveConfigurationError(
                 "WEAVE_WANDB_COOKIE should not be set in public mode."
             )
-        if os.path.exists(os.path.expanduser("~/.netrc")):
-            raise errors.WeaveConfigurationError(
-                "Please delete ~/.netrc while using WEAVE_WANDB_COOKIE to avoid using your credentials"
-            )
+        for netrc_file in ("~/.netrc", "~/_netrc"):
+            if os.path.exists(os.path.expanduser(netrc_file)):
+                raise errors.WeaveConfigurationError(
+                    f"Please delete {netrc_file} while using WEAVE_WANDB_COOKIE to avoid using your credentials"
+                )
     return cookie
 
 
+def weave_impersonated_username_cookie() -> typing.Optional[str]:
+    return os.environ.get("WEAVE_IMPERSONATED_USERNAME_COOKIE")
```

Comment:
this is a misnomer, since it is not a cookie, correct?

Reply:
It is specified as a cookie - see the change on line 78 of weave/wandb_api.py

But it's also an unwieldy name - I add the `_COOKIE` suffix for consistency with `WEAVE_WANDB_COOKIE` but I'd be fine with just calling it `WEAVE_IMPERSONATED_USERNAME` too.

Reply:
oh - i see the pattern. This makes sense now - thank you